Classes/New Issue/GithubClient+Template.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  GithubClient+Template.swift
+//  Freetime
+//
+//  Created by Ehud Adler on 11/11/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import GitHubAPI
+import GitHubSession
+import Squawk
+
+private let githubIssueURL = ".github/ISSUE_TEMPLATE"
+let tempalateRegex = ".github/ISSUE_TEMPLATE"
+
+struct IssueTemplateRepoDetails {
+    let owner: String
+    let name: String
+    let defaultBranch: String
+}
+
+extension GithubClient {
+
+    private func fetchTemplateFile(
+        repoDetails: IssueTemplateRepoDetails,
+        filename: String,
+        testingFile: String? = nil,
+        completion: @escaping (Result<String>) -> Void
+        ) {
+
+        // For Testing.
+        if let testingFile = testingFile { completion(.success(testingFile)) }
+
+        fetchFile(
+            owner: repoDetails.owner,
+            repo: repoDetails.name,
+            branch: repoDetails.defaultBranch,
+            path: "\(githubIssueURL)/\(filename)") { result in
+                switch result {
+                case .success(let file): completion(.success(file))
+                case .error(let error): completion(.error(error))
+                case .nonUTF8: completion(.error(nil))
+                }
+        }
+    }
+
+    func createTemplate(
```

Comment:
nit: this needs a new name, I keep thinking that it's actually creating something. Can it just use the "fetch" naming used elsewhere?